.github/workflows/antsibull-build.yml

```diff
@@ -54,6 +54,8 @@ jobs:
         run: |
           python3 -m pip install --upgrade pip
           python3 -m pip install poetry ansible-core
+          poetry install
+          poetry update
```

Comment:
Was the plan not to get rid of poetry entirely ? Should this be a pip install ``path/to/antsibull/src`` instead of poetry ?

Reply:
No, not at all! The plan is to get rid of poetry **in the role**. There is no reason why we should force users of this role to use poetry. poetry is a dependency management and build tool, not a "everyone has to use this to run the Python program you want to ship" tool.

Reply:
Works for me.